api/envoy/config/accesslog/v4alpha/accesslog.proto

```diff
@@ -240,6 +240,7 @@ message ResponseFlagFilter {
         in: "IH"
         in: "DPE"
         in: "UMSDR"
+        in: "RFCF"
```

Comment:
Any thoughts on extensible response flags for extensions? How many filters in the future will want to set response flags?

Reply:
We may want to eventually, but given this has to be used in the main repo (HCM logic has to know about this to handle the date header properly) I thing having it in core config is the right call.